extensions/wikia/Wall/WallThread.class.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
 class WallThread {
+	const DB_LIMIT_REPLIES = 500;
```

Comment:
Does this mean 500 comments do get loaded on the page? If so, is Cristina fine with this number?

Nitpicking here about name of the constant; how about `COMMENT_REPLY_LIMIT`?